docs/config-scheme.md

```diff
@@ -0,0 +1,370 @@
+
+## Installing react-native-config
+
+Install the package
+
+```bash
+// yarn
+yarn add react-native-config
+
+// npm
+npm install react-native-config --save
+```
+
+For iOS also run **pod install** after package is installed
+
+And below line of code to `android/app/build.gradle` to apply plugin
+
+```bash
+apply plugin: "com.android.application"
+apply from: project(':react-native-config').projectDir.getPath() + "/dotenv.gradle" // <- add this line
+```
+
+## Create .env files for each configuration
+
+`.env.development`
+
+```bash
+ENV=development
+API_URL=https://api.dev.com
+```
+
+`.env.staging`
+
+```bash
+ENV=staging
+API_URL=https://api.staging.com
+```
+
+`.env.staging`
+
+```bash
+ENV=production
+API_URL=https://api.com
+```
+
+## Setup for Android
+
+Now we need to define **envConfigFiles** in `build.gradle` associating builds with env files. To achieve this, add the below code before the apply from call, and be sure to leave the build cases in lowercase.
+
+`android/app/build.gradle`
+
+```bash
+// add this block
+project.ext.envConfigFiles = [
+   development: ".env.development",
+   staging: ".env.staging",
+   production: ".env.production",
+]
+// ---
+apply from: project(':react-native-config').projectDir.getPath() + "/dotenv.gradle"
+```
+
+Adding Product Flavor on project below line **compileSdkVersion**
+
+`android/app/build.gradle`
+
+```bash
+android {
+    ndkVersion rootProject.ext.ndkVersion
+    compileSdkVersion rootProject.ext.compileSdkVersion
+
+    // add this block
+    flavorDimensions "default"
+     productFlavors {
+        development {}
+        staging {}
+        production {}
+    }
+
+    defaultConfig {
+        applicationId project.env.get("APP_ID") // using env
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 1
+        versionName "1.0"
+        buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()
+        resValue "string", "build_config_package", "com.saigontechnolgy.rnbaseproject" // add this line
+
+        if (isNewArchitectureEnabled()) {
+            // We configure the CMake build only if you decide to opt-in for the New Architecture.
+            externalNativeBuild {
+                cmake {
+                    arguments "-DPROJECT_BUILD_DIR=$buildDir",
+                        "-DREACT_ANDROID_DIR=$rootDir/../node_modules/react-native/ReactAndroid",
+                        "-DREACT_ANDROID_BUILD_DIR=$rootDir/../node_modules/react-native/ReactAndroid/build",
+                        "-DNODE_MODULES_DIR=$rootDir/../node_modules",
+                        "-DANDROID_STL=c++_shared"
+                }
+            }
+            if (!enableSeparateBuildPerCPUArchitecture) {
+                ndk {
+                    abiFilters (*reactNativeArchitectures())
+                }
+            }
+        }
+    }
+
+   // ---
+...
+```
+
+Value `APP_ID` in env below:
```

Comment:
update docs: refer to chore: update docs scheme #90 (rn-base-project)

Reply:
Updated